### DISCUSSION TITLE
https://someonewhocares.org/hosts/hosts

### DISCUSSION THREAD

**Discussion by eraldinho** [HotCakeX/Harden-Windows-Security#356] [General]
dan pollock is doing a great job with his hosts file which is updated really often. I wonder if the use of dan pollock's hosts file could increase the system hardening if all Harden-Windows-Security features are already applied. any advice?

Thanks,

erald

**Comment by HotCakeX** [comment]
I think you're better off using online services such as [Serverless DNS](https://github.com/serverless-dns/serverless-dns) or NextDNS. Using a too big of a hosts file in Windows *could* lead to increased processor usage or latency. Those services I named allow you to use millions of domains and many different domain sources, they also automatically keep the domain sources up to date.

Check out this repo
https://github.com/HotCakeX/MicrosoftDomains

**Reply by eraldinho** [reply]
thanks for the advice, I will take a look at this repo
